packages/postcss/README.md

```diff
@@ -79,7 +79,32 @@ For more information on that topic, please have a look at [the documentation of
 
 #### Preset Options
 
-This preset has no preset configuration options.
+| Name | Type | Default | Required | Description |
+| --- | --- | --- | --- | --- |
+| `postcss.namedExport` | `Boolean` | `false` | _no_ | Whether CSS Modules do a named exports of class names rather than a default export of the class name object. |
+
+##### `postcss.namedExport`
+
+This option enables you to opt into CSS Modules exporting named exports, instead of doing a default export of the object, that holds all the class names.
+
+To enable it, set the following option in your Hops config.
+
+```json
+"hops": {
+  "postcss": {
+    "namedExport": true
+  }
+}
+```
+
+Then change your CSS Module imports to do namespace-imports.
+
+```diff
+- import styles from './module.css';
++ import * as styles from './module.css';
```

Comment:
Shouldn't we prefer `import { myClass, otherClass } from './module.css';` instead of importing everything?
We can mention that, to restore the previous behaviour, you can do `import * as`, but the way forward should be to import only what you need, I guess?

Reply:
I'll mention named imports.